floaters/src/main/java/com/mdg/droiders/floaters/CollapsedWindow.java

```diff
@@ -231,7 +231,7 @@ public boolean onTouch(View v, MotionEvent event) {
                         if (diffX < 10 && diffY < 10) {
                             mListener.clickHappened();
                         }
-
+                        mClosingButtonView.setVisibility(View.GONE);
```

Comment:
\[Checkstyle\] WARNING: 'block' child have incorrect indentation level 24, expected level should be one of the following: 16, 18, 20\.